.env.example

```
@@ -0,0 +1,2 @@
+GIT_USERNAME=your_github_username
+GIT_PASSWORD=your_github_personal_access_token
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)